src/interfaces/multipliers/IMultiplier.sol

```diff
@@ -2,7 +2,11 @@
 pragma solidity ^0.8.0;
 
 interface IMultiplier {
```

Comment:
I actually might be a good idea to just add `BASE_MULTIPLIER` constant here

Reply:
oh weird I didnt know that. I do think this makes things a little easier to work with so that the value doesn't need to be explicitly defined each time